build.gradle

```diff
@@ -26,7 +26,7 @@ plugins {
   id 'de.thetaphi.forbiddenapis' version '3.9' apply false
   id 'de.undercouch.download' version '5.6.0' apply false
   id 'net.ltgt.errorprone' version '3.1.0' apply false
-  id 'com.diffplug.spotless' version '6.5.2' apply false
+  id 'com.diffplug.spotless' version '7.2.1' apply false
```

Comment:
`main` branch is on 7.2.1 already: https://github.com/apache/solr/blob/80fa70f2ccf2a4fd03658d822a5ffab35819119f/gradle/libs.versions.toml#L80